samples/teams-sso-node/readme.md

```diff
@@ -122,49 +122,42 @@ Your tab needs to run as a registered Azure AD application to obtain an access t
     - 1fec8e78-bce4-4aaf-ab1b-5451cc387264
 ## Creating a Teams App
 
-Now you can use [App Studio](https://docs.microsoft.com/en-us/microsoftteams/platform/get-started/get-started-app-studio) app from within the Microsoft Teams client to help create your app manifest. If you do not have App studio installed in Teams, select Apps Store App at the bottom-left corner of the Teams app, and search for App Studio. Once you find the tile, select it and choose install in the pop-up window dialog box.
+Now you can use the [Developer Portal for Teams](https://learn.microsoft.com/microsoftteams/platform/concepts/build-and-test/teams-developer-portal) to configure, distribute and manage your application. You can access the [Developer Portal for Teams in a web browser](https://dev.teams.microsoft.com/) or as a [Teams App](https://teams.microsoft.com/l/app/14072831-8a2a-4f76-9294-057bf0b42a68).
 
-1. Open App Studio and select the **Manifest editor** tab.
+You can also use the [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) in **Visual Studio Code** to quickly create and deploy your Teams app.
 
-1. Choose the **Create a new app** tile. This will bring you into the **App Details** section.
+To use the Developer Portal for Teams:
 
-1. Under **App names** fill in the short name and full name as you wish
+1. Open [Developer Portal for Teams](https://dev.teams.microsoft.com/).
 
-1. Under **Identification** press **Generate** to generate an App Id (this is only for the Teams App). Then fill in
-    - **Package name:** `com.mgt.teamsSsoSample`
-    - **Version:** `1.0.0`
+2. Choose **Apps** from the left sidebar menu.
 
-1. Under **Descriptions** fill in the short and full description as you wish
+3. Click on **+ New app** from the options at the top left of the screen. This will open a modal where you enter an application name. Click **Add** to save it.
 
-1. Under **Developer information** section fill out your details
+4. Fill in the remaining required basic information of the application. Required fields have an asterik (*) after their titles.
+    > **Note**: The application ID is automatically created for you when you add the new application.
 
-1. Under **App URLs** provide links, ex:
-    - **Privacy statement** `https://www.microsoft.com/privacy`
-    - **Terms of use** `https://www.microsoft.com/termsofuse`
+    a. For **App names**, the short name will be the name you entered before for your application.
 
-1. In the left navigation, in the **Capabilities** section, select **Tabs**
+    b. For **Descriptions**, fill the **Short description** and **Long description**.
 
-1. Select **Add** to create a **Personal tab**
+    c. For **Developer**, fill a **Developer of company name**. Set the ngrok URL as the **Website**.
```

Comment:
Let's not assume ngrok. Should we just say : 

```suggestion
    c. For **Developer**, fill a **Developer of company name**. Set the the application URL as the **Website**.
```

Reply:
We might be using ngrok in this instance, but in the steps, I would just mark them as "application URL" so that way if they want to host their apps, it's still relevant. Makes sense? @musale